src/lib.rs

```diff
@@ -309,6 +311,7 @@ pub mod async {
 /// - redirect limit was exhausted
 pub fn get<T: IntoUrl>(url: T) -> ::Result<Response> {
     Client::builder()
+        .use_sys_proxy()
```

Comment:
This top-level `get` should likely just use all the defaults of the builder.

Reply:
I have move it into `Client::builder()` rather than `ClientBuilder::new()`.

I'm not very sure where is `use_sys_proxy()` call should be put.

But I think that move it into `Client::builer()` is better, because `ClientBuilder::new()` is too low level.  And the `ClientBuilder::new()` constructor shouldn't do anything more than initialize `ClientBuilder` object.